kratos/tests/materials.json

```diff
@@ -4,9 +4,6 @@
 		"properties_id": 1,
 		"Material": {
 			"name": "steel",
-			"constitutive_law": {
-				"name": "LinearElastic3DLaw"
```

Comment:
Why do you remove this?
I think this is an important check

Or does the utility not support this?

Reply:
Huuuum, @marandra after your string clean up does it work?

Reply:
This change wasn't intended for the PR, I was just testing @loumalouomega change to allow for missing constitutive law. However, I do think we should update the testfile to test it.

Reply:
just reverted it to the original version